k8s/aat/common/sscs/sscs-tya-notif.yaml

```diff
@@ -28,7 +28,6 @@ spec:
         HOURS_START_TIME: 0
         HOURS_END_TIME: 17
         COVID_19_NOTIFICATION_FEATURE: false
-        UC_ENABLED: true
```

Comment:
I think this was removed in the code, is that correct?

Reply:
Yes